app/src/main/res/layout/fragment_waiting_submissions.xml

```diff
@@ -0,0 +1,52 @@
+<FrameLayout
```

Comment:
You're totally right! I forgot I could add a gravity property to the outmost layout to center this content!
Fixed now.